### PR TITLE
Scrollbar improvements

### LINKED ISSUE #11190: Make layer panel scrollbar wider
### Context
When not using a trackpad or a mouse scroll wheel, it is very hard to scroll the layers. Also, it doesn't appear until scrolling which is very unintuitive

### LINKED ISSUE #11194: Infinite scroll on horizontal widget list on builder
### Steps to Reproduce

1. Create a map
2. Make your browser window small (< 1200px width) so the widget layout goes to bottom
3. Add a lot of widgets (the minimum I've been able to reproduce is 6)
4. Scroll to the right on the widget list

### Current Result
No scrolling limit

screenshot_20170105_125915
 
### Expected result
Scrolling stops after the last widget

### Browser and version
Firefox and Chrome on Linux

cc @xavijam (the trick is in the number of widgets, minimum of 6 to reproduce)

### PR DESCRIPTION
**Pull request by nobuti**
PR fixes #11190.
PR fixes #11194.

This PR includes an update in the Perfect Scrollbar dependency. You can see the proper update in https://github.com/nobuti/perfect-scrollbar/pull/1. Also, this allows us to remove the polling in the scroll view to check when new children are appended to the container in order to update the scroll. That is delegated to the plugin now.

Also, check that #11194 is fixed with these improvements.

### CODE REVIEW

File: app/assets/stylesheets/editor-3/editor.css.scss [hunk 1]
```diff
@@ -213,14 +216,14 @@
...
     }
-    .CDB-Widget-canvas .hasXScroll>.ps-scrollbar-x-rail {
-      display: block;
-      width: 100%;
-      opacity: 0.6;
-    }
-    .CDB-Widget-canvas .hasXScroll>.ps-scrollbar-x-rail>.ps-scrollbar-x {
-      min-width: 50%;
-    }
+    // .CDB-Widget-canvas .hasXScroll>.ps-scrollbar-x-rail {
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/editor.css.scss]: I'd remove the CSS if it is not necessary anymore.

**Comment by xavijam** [comment]
@piensaenpixel, could you point me which new things I should notice in the UI?

**Comment by xavijam** [comment]
Ok, scrollbar always visible, I like it 👍 . One thing I've notice is:

screen shot 2017-01-19 at 14 17 23

Wouldn't you separate the scrollbar at least a pixel to the edge of the content?

**Comment by nobuti** [comment]
Test if this fixes https://github.com/CartoDB/cartodb/issues/11194, please

**Comment by xavijam** [comment]
Yeah, it is fixed.

**Comment by xavijam** [comment]
Talking with @piensaenpixel offline, rest of the designers said that scrollbar is ok where it appears, so 👍 .

**Comment by piensaenpixel** [comment]
I'm agree with you but it is an @urbanphes and @josecruz decision...

**Pull request merged**